README.md

```diff
@@ -25,7 +25,7 @@ _Have another more specific idea? You may want to check out our vibrant collecti
 
 1.  **Start developing.**
 
-    Navigate into your new site’s directory and start it up.
```

Comment:
Is this some find/replace junk?

Reply:
I've not done anything on the readme: i think it's just part of the default page? i'm not sure

Reply:
looking at it, probably an accidental replace while i was looking for the keyword

Reply:
This kind of mistake can be avoided by only intentionally adding files when committing.

E.g. don't use `git commit .`.  It's better to run `git status` to review the changes you want to commit (use `git diff <file>), and then add each file you want to commit.  That way if there's an unintentional edit, like the one here, it never gets committed.